Storage/Storage/CoreData/CoreDataIterativeMigrator.swift

```diff
@@ -144,20 +144,34 @@ private extension CoreDataIterativeMigrator {
         return tempDestinationURL
     }
 
-    /// Move the original source store to a backup location.
+    /// Move the original source store files to a backup location (typically /Documents/backup).
     ///
-    func makeBackup(at storeURL: URL) throws -> URL {
-        let backupURL = storeURL.deletingLastPathComponent().appendingPathComponent("backup")
-        try? fileManager.removeItem(at: backupURL)
-        try? fileManager.createDirectory(atPath: backupURL.path, withIntermediateDirectories: false, attributes: nil)
+    /// The files that will be moved are:
+    ///
+    /// - {store_filename}.sqlite
+    /// - {store_filename}.sqlite-wal
+    /// - {store_filename}.sqlite-shm
+    ///
+    /// Where {store_filename} is most probably "WooCommerce".
+    ///
+    func moveStoreFilesToBackupFolder(storeURL: URL) throws -> URL {
+        let storeFolderURL = storeURL.deletingLastPathComponent()
+        let backupURL = storeFolderURL.appendingPathComponent("backup")
+
+        if fileManager.fileExists(atPath: backupURL.path) {
+            try fileManager.removeItem(at: backupURL)
+        }
+        try fileManager.createDirectory(at: backupURL, withIntermediateDirectories: false, attributes: nil)
```

Comment:
Same here, I think it needs to be `try?` to prevent stopping of execution.

Reply:
Responded in [the thread](https://github.com/woocommerce/woocommerce-ios/pull/3069/files#r513120209) above . 🙂

Reply:
@shiki What happens if the backup folder exists from a previous migration? Will this fail?

Reply:
The backup folder is deleted right before this line is executed so this should pass. 

```swift
        if fileManager.fileExists(atPath: backupURL.path) {
            try fileManager.removeItem(at: backupURL)
        }
```

But this is no longer relevant since I removed the backup logic. 🙂